anvil/src/eth/error.rs

```diff
@@ -72,6 +72,8 @@ pub enum BlockchainError {
     TimestampError(String),
     #[error(transparent)]
     DatabaseError(#[from] DatabaseError),
+    #[error("EIP-1559 style fee params (maxFeePerGas or maxPriorityFeePerGas) received but they are not supported by the current hardfork.\n\nYou can use them by running anvil with '--hardfork londong' or later.")]
+    EIP1559TransactionUnsupportedAtHardfork,
 }
```

Comment:
Missing `EIP2930TransactionUnsupportedAtHardfork`. The error message can be something like 
```
Access list received but is not supported by the current hardfork.\n\nYou can use it by running 
anvil with '--hardfork berlin' or later.
```